.goreleaser.yaml

```diff
@@ -107,6 +107,34 @@ docker_signs:
     - '${artifact}'
     - "--yes" # needed on cosign 2.0.0+
 
+brews:
+  - tap:
+      owner: openfga
+      name: homebrew-tap
+      pull_request:
+        enabled: true
+    homepage: "https://openfga.dev/"
+    description: "A high performance and flexible authorization/permission engine built for developers and inspired by Google Zanzibar."
+    license: "Apache-2.0"
+    folder: "Formula"
+    url_template: "https://github.com/openfga/openfga/releases/download/{{ .Tag }}/{{ .ArtifactName }}"
+    download_strategy: CurlDownloadStrategy
+
+    # update the head formula on each release
+    custom_block: |
+      head "https://github.com/openfga/cli.git", :branch => "main"
+    dependencies:
```

Comment:
I don't really understand why go is optional but git is required. In case we are downloading the binary, none are required and in the case we are building from source, both should be required no?